src/activities/Elsa.Activities.Email/Activities/SendEmail/SendEmail.cs

```diff
@@ -23,8 +25,14 @@ public SendEmail(ISmtpService smtpService)
         [ActivityProperty(Hint = "The sender's email address.")]
         public string From { get; set; }
 
-        [ActivityProperty(Hint = "The recipient's email address.")]
-        public string To { get; set; }
+        [ActivityProperty(Hint = "The recipients email addresses.")]
```

Comment:
Didn't we already merge the send email PR?

Reply:
Yes, that should also not be in the pull request